src/Classes/Cli/HelpRenderer.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of MMLC - ModifiedModuleLoaderClient.
+ *
+ * (c) Robin Wieschendorf <mail@robinwieschendorf.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RobinTheHood\ModifiedModuleLoaderClient\Cli;
+
+class HelpRenderer
+{
+    private const INDENT = '  ';
+    private string $description = '';
+    private array $arguments = [];
```

Comment:
I think it would be better if we called them Sections. I find it confusing that there is an argument called Argument and one called Option.